### ISSUE TITLE
When discovered through 'find', bad symlinks are not handled properly

### ISSUE THREAD

**Issue by PanosFirmpas** [sahib/rmlint#398]
rmlint doesn't seem to have a problem with bad symlinks when it does file discovery itself,       
but it ~~crashes~~ exits elegantly with an error when given a bad symlink through find.       
           
Here's some quick commands to recreate the problem:         

```bash
echo "test" > testfile.txt
ln -s testfile.txt link1.txt
rm testfile.txt

# Now, this works perfectly fine
rmlint
# this gives an error
find ./ | rmlint -
# so does this
find ./ -print0 | rmlint -0
```

Am I missing something?

**Comment by sahib** [comment]
Hello @PanosFirmpas,

> Am I missing something?

Nope.

That's actually unfortunate and should be fixed. It would not work also if you would do ``rmlint link1.txt``. It should work now better as of 5ca6d25905cd372e8b07ecd9696a097acb0d5742, but it will still fail if all paths consist of bad symbolic links.

**Comment by sahib** [comment]
The situation should be fixed as of 6853ee3b969cd59379b97b493ebbd63673b1dae6 (i.e. ``develop`` branch). Please check back and see if it works out for you.
